source/module_lr/utils/lr_util.cpp

```diff
@@ -101,88 +105,100 @@ namespace LR_Util
     {
         container::Tensor t(DAT::DT_COMPLEX_DOUBLE, DEV::CpuDevice, { m.nr, m.nc });
-        for (int i = 0;i < t.NumElements();++i) {t.data<std::complex<double>>()[i] = m.c[i];
-}
+        for (int i = 0;i < t.NumElements();++i) {
+            t.data<std::complex<double>>()[i] = m.c[i];
+        }
         return t;
     }
     std::vector<container::Tensor> mat2ten_complex(std::vector<ModuleBase::ComplexMatrix>& m)
     {
         std::vector<container::Tensor> t;
-        for (int i = 0;i < m.size();++i) { t.push_back(mat2ten_complex(m[i]));
-}
+        for (int i = 0;i < m.size();++i) {
+            t.push_back(mat2ten_complex(m[i]));
+        }
         return t;
     }
     ModuleBase::ComplexMatrix ten2mat_complex(container::Tensor& t)
     {
         ModuleBase::ComplexMatrix m(t.shape().dims()[0], t.shape().dims()[1]);
-        for (int i = 0;i < t.NumElements();++i) {m.c[i] = t.data<std::complex<double>>()[i];
-}
+        for (int i = 0;i < t.NumElements();++i) {
+            m.c[i] = t.data<std::complex<double>>()[i];
+        }
         return m;
     }
     std::vector<ModuleBase::ComplexMatrix> ten2mat_complex(std::vector<container::Tensor>& t)
     {
         std::vector<ModuleBase::ComplexMatrix> m;
-        for (int i = 0;i < t.size();++i) { m.push_back(ten2mat_complex(t[i]));
-}
+        for (int i = 0;i < t.size();++i) {
+            m.push_back(ten2mat_complex(t[i]));
+        }
         return m;
     }
 
     ModuleBase::matrix vec2mat(const std::vector<double>& v, const int nr, const int nc)
     {
         assert(v.size() == nr * nc);
         ModuleBase::matrix m(nr, nc, false);
-        for (int i = 0;i < v.size();++i) { m.c[i] = v[i];
-}
+        for (int i = 0;i < v.size();++i) {
+            m.c[i] = v[i];
+        }
         return m;
     }
     ModuleBase::ComplexMatrix vec2mat(const std::vector<std::complex<double>>& v, const int nr, const int nc)
     {
         assert(v.size() == nr * nc);
         ModuleBase::ComplexMatrix m(nr, nc, false);
-        for (int i = 0;i < v.size();++i) { m.c[i] = v[i];
-}
+        for (int i = 0;i < v.size();++i) {
+            m.c[i] = v[i];
+        }
         return m;
     }
     std::vector<ModuleBase::matrix> vec2mat(const std::vector<std::vector<double>>& v, const int nr, const int nc)
     {
         std::vector<ModuleBase::matrix> m(v.size());
-        for (int i = 0;i < v.size();++i) { m[i] = vec2mat(v[i], nr, nc);
-}
+        for (int i = 0;i < v.size();++i) {
+            m[i] = vec2mat(v[i], nr, nc);
+        }
         return m;
     }
     std::vector<ModuleBase::ComplexMatrix> vec2mat(const std::vector<std::vector<std::complex<double>>>& v, const int nr, const int nc)
     {
         std::vector<ModuleBase::ComplexMatrix> m(v.size());
-        for (int i = 0;i < v.size();++i) { m[i] = vec2mat(v[i], nr, nc);
-}
+        for (int i = 0;i < v.size();++i) {
+            m[i] = vec2mat(v[i], nr, nc);
+        }
         return m;
     }
```

Comment:
similarly, if from the nested std::vector can construct a matrix, then it can be one constructor of matrix, and vice versa if want to convert matrix to nested or flattened std::vector, then it should be interface implemented in matrix

Reply:
Thanks for reminding me that ... they're legacy interfaces. I've deleted them in the latest commit.
Now module_lr does not use `matrix` and `ComplexMatrix` any more (and I expect these two classes will finally be eliminated in the whole ABACUS code).